tools/devops/cleanup.sh

```diff
@@ -0,0 +1,7 @@
+sudo find  $(dirname $(dirname "$(System.DefaultWorkingDirectory)")) -type d -name 'r*' -not -name $(basename $(dirname "$(System.DefaultWorkingDirectory)")) -maxdepth 1 -exec rm -rf {} +
```

Comment:
What's this line supposed to find and delete?

Reply:
First time I read it I thought it was just the source folder but `-name 'r*'`?? I'm not sure so I'm deleting it. I'll ask the person who sent it to me but I think what they really wanted gone was VSM.

Reply:
It also doesn't work as is because of the environment variable, I'm not attached to this (: